src/views/changesets_list.js

```diff
@@ -185,31 +186,29 @@ class ChangesetsList extends React.PureComponent {
             <Button className="mx3">Filters</Button>
           </NavLink>
         </header>
-        <header
-          className={`border-l border-b border-b--1 border-l--4 border-color-neutral px12 py3 ${diff >
-            0
-            ? 'bg-yellow-faint'
-            : 'bg-gray-faint'} flex-child align-items--center`}
-        >
+        <header className="px12 py6 bg-darken10 flex-child align-items--center">
           <span className="flex-parent flex-parent--row justify--space-between color-gray txt-s txt-bold">
             <span>
-              Results:
+              {this.props.currentPage && this.props.currentPage.get('count')
+                ? numberWithCommas(this.props.currentPage.get('count'))
+                : 0}
               {' '}
-              {this.props.currentPage && this.props.currentPage.get('count')}
+              changesets.
             </span>
             <span className="flex-parent flex-parent--row">
-              {diffLoading && <span className="loading loading--s inline" />}
-              <span
+              <span className="px6">
+                {diffLoading && <span className="loading loading--s inline" />}
+                {diff > 0 ? `${diff} new` : ''}
+              </span>
+              <button
                 onClick={this.reloadCurrentPage}
-                className={`pointer ${diff > 0
-                  ? 'bg-yellow-light-on-hover'
-                  : 'bg-gray-light-on-hover'} round`}
+                className="btn btn--s color-gray border border--gray round bg-gray-faint bg-white-on-hover"
               >
-                {diff > 0 ? `${diff} New` : ''}
-                <svg className="icon inline-block align-middle ">
-                  <use xlinkHref="#icon-rotate" />
+                Refresh
+                <svg className="icon icon--s inline-block align-middle ">
+                  <use xlinkHref="#icon-refresh" />
```

Comment:
Do we really need to be verbose about refresh.
I feel it is better to just use the icon.